### PR TITLE
feat: add tips for mobile forms

### PR DESCRIPTION
**Pull request by lorenamelor**
Tips for using attributes that make the usability of the mobile form more pleasurable, such as:

- Focus on the next input or submit on a form using the keyboard;
- Display the correct keyboard according to the type of the form;
- Make the input already fill words with the first letter in uppercase.

### CODE REVIEW

**Review by RaphaelOliveiraMoura — APPROVED** [review]

**Review by HigorAlves — COMMENTED** [review]
I think we can add 2 more things:
Input type and a default message error for invalid data.

File: Mobile-Development-and-React-Native.md [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+	    id="firstName"
+	    value={firstName}
+	    onChange={handleOnChange}
+	    returnKeyType="next"
+	    onSubmitEditing={() => inputRefLastName.current.focus()}
+    />
+    <TextInput
+	    id="lastName"
+	    ref={inputRefLastName}
+		value={name}
```
Comment (matAlmeida) [inline Mobile-Development-and-React-Native.md:58]: ↹

File: Mobile-Development-and-React-Native.md [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
-*@CONTRIBUTE Please teach me the ways of RN, master...*
\ No newline at end of file
+## Forms
+
+Forms are one of the most annoying parts for users, so the more we can turn the experience of filling them out into something good, the less discomfort will be caused. Therefore, some react-native resources used for these purposes will be highlighted.
+
+
+### Focus on the next input and submit directly from the keyboard
+
+It is very annoying when we are filling out a form on the cell phone and when we finish a text field we have to scroll down and click on the next one. For this reason, keyboards have a key that can assist in this action. It is that key that is in the lower right corner of the keyboards. For her we can define a function (such as to focus on the next text field or send the form) and even a nomenclature that best suits the situation.
```
Comment (matAlmeida) [inline Mobile-Development-and-React-Native.md]: Just some change suggestion for better clarity and to match names used on other parts of the text, but would require the review of more people:

```suggestion
It is very annoying when we are filling out a form on the cell phone and we need to scroll down and select the next input. For this reason, mobile keyboards have a return key to assist in this case. The return key is usually found at the lower right corner of the keyboards and we can pass a function to be executed when it's pressed, such as to focus on the next text field or submit the form. Some platforms even allow you to replace it's label for best suit your use case.
```

**Review by matAlmeida — COMMENTED** [review]
👽

**Review by HigorAlves — APPROVED** [review]

**Review by philipeatela — APPROVED** [review]

**Pull request merged**